src/content/service.js

```diff
@@ -15,53 +16,94 @@ const getShortUid = () =>
     .join('')
     .substr(0, 6)
 
-module.exports = ({ db, botfile, projectLocation, logger }) => {
-  let categories = []
+const getNewItemId = category => {
+  const prefix = (category.ummBloc || category.id).replace(/^#/, '')
+  return `${prefix}-${getShortUid()}`
+}
 
-  const scanAndRegisterCategories = async () => {
-    categories = []
+const prepareDb = async () => {
+  const knex = getInMemoryDb()
+
+  await knex.schema.createTable('content_items', table => {
+    table.string('id').primary()
+    table.text('data')
+    table.text('formData')
+    table.text('metadata')
+    table.string('categoryId')
+    table.text('previewText')
+    table.string('createdBy')
+    table.timestamp('createdOn')
+  })
+
+  return knex
+}
 
-    const formDir = path.resolve(projectLocation, botfile.formsDir || './content/forms')
+module.exports = async ({ botfile, projectLocation, logger }) => {
+  const categories = []
+  const categoryById = {}
+  const fileById = {}
 
-    if (!fs.existsSync(formDir)) {
-      return categories
+  const formDir = path.resolve(projectLocation, botfile.formsDir || './content/forms')
```

Comment:
Let's change the default path to being `./forms`. I meant to change it last time but forgot